app/Http/Controllers/StudentsCouncil/AnonymousQuestionController.php

```diff
@@ -37,11 +35,25 @@ public function __construct()
      * the list of questions
      * and the option to add new ones.
      */
-    public function indexSemesters()
+    public function index()
     {
         $this->authorize('administer', AnswerSheet::class);
 
-        return view('student-council.anonymous-questions.index_semesters');
+        return view('student-council.anonymous-questions.index');
+    }
+
+    /**
+     * Whether we allow a semester to be added questions.
+     */
+    public function canAddQuestionTo(Semester $semester): bool
+    {
+        // true for future, false for past semesters
+        if (!$semester->isCurrent()) {
+            return !$semester->isClosed();
+        } else {
+            $endDate = $this->getEndDate();
+            return is_null($endDate) || !$endDate->isPast();
+        }
```

Comment:
What about this?

Reply:
Creating a new question should create it to the current semester evaluation that is set based on the periodic event (regardless if it is active or not). In the UI, make it clear that the questions are going to be created to the semester that is selected on the periodic event. 
And you don't need this policy.

You don't need to map any post request to a specific semester. 
The older semester's results can be viewed, that's all.

I'm sorry if it is still confusing for you, maybe we can have a meeting to discuss this further and identify what should we do to make this clearer in the future.

Reply:
Am I right that this means some kind of simplification so that we do not need to refer to semesters? I don't insist on being able to edit every future semester; so if you would like, I can point that out in the UI as you've suggested

Reply:
Yes, that's what I'm trying to say

Reply:
Okay:) I'll move forward with it.